dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3554,30 +3511,9 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.MASTER)
           .build();
-  public static final PropertyKey MASTER_TIERED_STORE_GLOBAL_LEVEL0_ALIAS =
-      stringBuilder(Name.MASTER_TIERED_STORE_GLOBAL_LEVEL0_ALIAS)
-          .setDefaultValue(Constants.MEDIUM_MEM)
-          .setDescription("The name of the highest storage tier in the entire system.")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
-          .setScope(Scope.MASTER)
-          .build();
-  public static final PropertyKey MASTER_TIERED_STORE_GLOBAL_LEVEL1_ALIAS =
-      stringBuilder(Name.MASTER_TIERED_STORE_GLOBAL_LEVEL1_ALIAS)
-          .setDefaultValue(Constants.MEDIUM_SSD)
-          .setDescription("The name of the second highest storage tier in the entire system.")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
-          .setScope(Scope.MASTER)
-          .build();
-  public static final PropertyKey MASTER_TIERED_STORE_GLOBAL_LEVEL2_ALIAS =
-      stringBuilder(Name.MASTER_TIERED_STORE_GLOBAL_LEVEL2_ALIAS)
-          .setDefaultValue(Constants.MEDIUM_HDD)
-          .setDescription("The name of the third highest storage tier in the entire system.")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
-          .setScope(Scope.MASTER)
-          .build();
   public static final PropertyKey MASTER_TIERED_STORE_GLOBAL_LEVELS =
       intBuilder(Name.MASTER_TIERED_STORE_GLOBAL_LEVELS)
-          .setDefaultValue(3)
+          .setDefaultValue(0)
```

Comment:
I'm not sure but this might break some tests. I see `BlockMaster` relies on this to create `DefaultStorageTierAssoc` and uses that for some parsing/logic.

Reply:
yes, I fixed some tests for it, like setting the level and add some default tiers to make current unit test run successfully. but we may eventually remove this tiered logic, right?